samples/bluetooth/unicast_audio_client/boards/native_posix.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_LOG_MODE_IMMEDIATE=y
+#Enable TINYCRYPT for native_posix
```

Comment:
I guess this line can be omitted given that it is in the `native_posix.conf` file :)